lib/shared/addon/oauth/service.js

```diff
@@ -16,15 +16,15 @@ export default Service.extend({
   authType:    '',
 
   generateState() {
-    return set(this, 'session.githubState', `${ Math.random() }`);
+    return set(this, 'session.state', `${ Math.random() }`);
```

Comment:
I worry that `state` is too general for a session variable name. What do you think about `oauthState` or `authState`? The actual name isn't that important to me just that we add some extra context to it.

Reply:
I agree, that makes much more sense.